db/system_keyspace.cc

```diff
@@ -1618,6 +1622,25 @@ schema_ptr system_keyspace::dicts() {
     return schema;
 }
 
+schema_ptr system_keyspace::view_building_tasks() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, VIEW_BUILDING_TASKS);
+        return schema_builder(NAME, VIEW_BUILDING_TASKS, std::make_optional(id))
+                .with_column("key", utf8_type, column_kind::partition_key)
+                .with_column("id", timeuuid_type, column_kind::clustering_key)
+                .with_column("type", utf8_type)
+                .with_column("state", utf8_type)
+                .with_column("base_id", uuid_type)
+                .with_column("view_id", uuid_type)
+                .with_column("last_token", long_type)
+                .with_column("host_id", uuid_type)
+                .with_column("shard", int32_type)
+                .with_hash_version()
```

Comment:
Does this table make sense with tablets? Tasks would move around as tablets are migrated. Shouldn't it be a column in system.tablets for those tables?

Reply:
> Shouldn't it be a column in system.tablets for those tables?

That's an alternative approach but we went with separate table for the view building tasks. In case of tablet migration/resize/RF change, the tasks are adjusted accordingly (commit bf90e38f5aecaf87fcbbaf1f2421d1aedac815ff (I've notice now that I need to improve the commit message a little bit)).